src/droplet.F

```diff
@@ -200,8 +210,7 @@ subroutine droplet_driver(dt,dbldt,mtime,xh,uh,ruh,xf,yh,vh,rvh,yf,zh,mh,rmh,zf,
 !     (for the parcel subroutines only!)
 
 #ifdef MPI
-      !$acc data copyout (pdata_neighbor) &
-      !$acc      copyin  (randomNumbers,randomNumbers%state)
+      !$acc data create(pdata_neighbor,msk)
```

Comment:
why we do not need `!$acc copyin (randomNumbers,randomNumbers%state)`?

Reply:
The randomNumbers is no longer used in the droplet_driver subroutine.  It has been moved to the droplet injection code.

Reply:
Gotcha!